example/lib/main.dart

```diff
@@ -191,11 +191,11 @@ class _MyHomePageState extends State<MyHomePage> {
                       title: Text('Delete'),
                       content: Text('Item will be deleted'),
                       actions: <Widget>[
-                        FlatButton(
+                        TextButton(
```

Comment:
**question:** was this part of the migration, no changes though?

Reply:
Yes that's part of the migration too! `FlatButton` is deprecated in Flutter 2.x